src/Isolation/Masa.BuildingBlocks.Isolation/Options/DbConnectionOptions.cs

```diff
@@ -0,0 +1,14 @@
+namespace Masa.BuildingBlocks.Isolation.Options;
+public class DbConnectionOptions
+{
+    public string TenantId { get; set; }
+
+    public string Environment { get; set; }
+
+    public string ConnectionString { get; set; }
+
+    /// <summary>
+    /// Used to control the configuration with the highest score when multiple configurations are satisfied. The default score is 100
+    /// </summary>
+    public int Score { get; set; } = 100;
```

Comment:
If there has the same score, what's the control logic?

Reply:
If the scores are the same, choose the first one in the order of the Isolations configuration